pkg/scraper/scraper.go

```diff
@@ -83,110 +77,145 @@ func RegisterScraperMetrics(registrationFunc func(metrics.Registerable) error) e
 	return nil
 }
 
-func NewScraper(nodeLister v1listers.NodeLister, client client.KubeletMetricsGetter, scrapeTimeout time.Duration) *scraper {
-	return &scraper{
-		nodeLister:    nodeLister,
-		kubeletClient: client,
-		scrapeTimeout: scrapeTimeout,
+func NewManageNodeScraper(client client.KubeletMetricsGetter, scrapeTimeout time.Duration, metricResolution time.Duration, store storage.Storage) *manageNodeScraper {
```

Comment:
Rename is not needed, I don't think we will want to have both implementations in the same codebase.

Reply:
ok, I will update it.